src/test/datascience/notebook/introStartPage.vscode.test.ts

```diff
@@ -90,6 +90,7 @@ suite('DataScience - VSCode Notebook - Native Notebook Experiment', function ()
         await sleep(1_000);
 
         assert.isUndefined(notebookEditorProvider.activeEditor);
+        notebookEditorProvider.editors.map(item => assert.fail(item.file.fsPath, undefined, 'There should be no document open'));
```

Comment:
Logging, for some reason these tests fail on CI, the `suitesetup` ensures that all files are closed, but this fails.